jenkins/model_output_processing_Jenkinsfile

```diff
@@ -102,22 +104,103 @@ pipeline {
         success {
             mail to: 'mhines@usgs.gov, wwatkins@usgs.gov',
             subject: "${TIER} Success: ${currentBuild.fullDisplayName}",
-            body: "Pipeline finished successfully ${env.BUILD_URL}"
+            body: """Pipeline finished successfully ${env.BUILD_URL}
```

Comment:
detailed results message with links to each variables test results, providing the order of magnitude file as a summary, and then links to each variables daily data output comparisons with the historic variable's 95%*1.5 max value, as well as the max for each variable for each HRU regardless of day.